### PR TITLE
Home page template

### PR DESCRIPTION
**Pull request by noahmanger**
Just wanted to get this out of the way. It needs the actual hero image eventually, but now it actually looks like a thing!

### CODE REVIEW

File: fec/fec/templates/base.html [hunk 1]
```diff
@@ -13,9 +13,12 @@
...
 
     {# Global stylesheets #}
-    {% compress css %}
+<!--     {% compress css %}
     <link rel="stylesheet" type="text/x-scss" href="{% static 'scss/fec.scss' %}">
-    {% endcompress %}
+    {% endcompress %} -->
+
+     <!-- Uncomment while developing to use css served from fec-style -->
+    <link rel="stylesheet" href="http://localhost:8080/css/styles.css" />
```
Comment (jmcarp) [inline fec/fec/templates/base.html]: What do you think about keying this on an environment variable or adding to `settings/local.py`? For example:
```
FEC_CMS_STYLE=http://localhost:8080/css/styles.css ./fec/manage.py runserver
```
or
```python
# local.py
FEC_CMS_STYLE = 'http://localhost:8080/css/styles.css'
```
(settings/local.py is a configuration file that isn't checked into git, and is used, appropriately enough, for settings specific to a specific local environment)

Comment (noahmanger) [inline fec/fec/templates/base.html]: Oops, yep. I was going to ask about something like this. I like that idea.

Comment (noahmanger) [inline fec/fec/templates/base.html]: We should also do this in the web-app too.

**Comment by noahmanger** [comment]
I removed the link to the local stylesheet for now. Wasn't sure how to set up the local variable.

**Pull request merged**